inc/msp/msp_id.hpp

```diff
@@ -1,166 +1,356 @@
 #ifndef MSP_ID_HPP
 #define MSP_ID_HPP
 
+#include <cstdint>
+
 namespace msp {
 
-enum class ID : uint8_t {
+enum class ID : uint16_t {
```

Comment:
Is it possible to move the IDs to the file with the message definitions and use a forward declaration for where the ID is used outside of messages?